marketplace/deployer_util/provision.py

```diff
@@ -356,17 +359,19 @@ def dns1123_name(name):
   fixed = re.sub(r'[.]', '-', fixed)
   fixed = re.sub(r'[^a-z0-9-]', '', fixed)
   fixed = fixed.strip('-')
-  if len(fixed) > 64:
-    fixed = fixed[:59]
+  fixed = limit_name(fixed)
```

Comment:
According to the [RFC](https://tools.ietf.org/html/rfc1123.html#page-13), this can be up to 255. It's mandatory that it can be up to 63. I'll say stick with 63 (and correct my earlier typo of 64) here, but change the default of `limit_name` to something larger, like 127.